tests/unit/tuner/test_onnx.py

```diff
@@ -0,0 +1,217 @@
+import tempfile
+from pathlib import Path
+
+import numpy as np
+import onnxruntime
+import paddle
+import pytest
+import torch
+from tensorflow import keras
+
+from finetuner.tuner.onnx import to_onnx, validate_onnx_export
+
+
+@pytest.fixture
+def get_pytorch_linear_model():
+    def _get_model(input_shape):
+        return torch.nn.Sequential(
+            torch.nn.Flatten(),
+            torch.nn.Linear(in_features=input_shape, out_features=64),
+            torch.nn.ReLU(),
+            torch.nn.Linear(in_features=64, out_features=32),
+        )
+
+    return _get_model
+
+
+@pytest.fixture
+def pytorch_conv_model():
+    return torch.nn.Sequential(
+        torch.nn.Conv2d(2, 3, 2, stride=1),
+        torch.nn.ReLU(),
+    )
+
+
+@pytest.fixture
+def pytorch_lstm_model():
+    return torch.nn.Sequential(torch.nn.LSTM(32, 2))
+
+
+@pytest.fixture
+def get_keras_linear_model():
+    def _get_model(input_dim):
+        return keras.Sequential(
+            [
+                keras.layers.Flatten(input_shape=(input_dim,)),
+                keras.layers.Dense(64, activation='relu'),
+                keras.layers.Dense(32),
+            ]
+        )
+
+    return _get_model
+
+
+@pytest.fixture
+def get_keras_conv_model():
+    def _get_model(input_shape):
+        return keras.Sequential(
+            [
+                keras.layers.Input(input_shape),
+                keras.layers.Conv2D(input_shape[-1], 3, strides=2, activation='relu'),
+            ]
+        )
+
+    return _get_model
+
+
+@pytest.fixture
+def get_keras_lstm_model():
+    def _get_model(input_shape):
+        return keras.Sequential(
+            [
+                keras.layers.Input(input_shape),
+                keras.layers.LSTM(4),
+            ]
+        )
+
+    return _get_model
+
+
+@pytest.fixture
+def get_paddle_linear_model():
+    def _get_model(input_dim):
+        return paddle.nn.Sequential(
+            paddle.nn.Flatten(),
+            paddle.nn.Linear(in_features=input_dim, out_features=64),
+            paddle.nn.ReLU(),
+            paddle.nn.Linear(in_features=64, out_features=32),
+        )
+
+    return _get_model
+
+
+@pytest.fixture
+def paddle_conv_model():
+    return paddle.nn.Sequential(paddle.nn.Conv2D(3, 3, 2, stride=1))
+
+
+@pytest.fixture
+def get_paddle_lstm_model():
+    def _get_model():
+        return paddle.nn.Sequential(paddle.nn.LSTM(32, 2))
+
+    return _get_model
```

Comment:
for all these fixtures, if you can copy fixtures from `test/unit/tailor/test_torch.py` (and keras, paddle), and move it into `conftest` will be even better. (feel free to not do it, but we need to clean this up)

Reply:
please ignore this comment, I'll create a common model fixture in projection head PR